ui/templates/includes/learning_resource.html

```diff
@@ -48,7 +48,12 @@
         <div class="tile-meta">
           <span class="meta-item">{{ result.course }}</span>
           <span class="meta-item">{{ result.run }}</span>
-          <span class="meta-item"><a href="{{ result.preview_url }}" target="_blank">Preview</a></span>
+          <span class="sub-meta">
+            <span class="meta-item mi-col-1"><i class="fa fa-eye"></i> {{ result.object.xa_nr_views }}</span>
```

Comment:
I was aware of the performance hit but I had assumed this wasn't in the search index, but since it is I'll use that. By the way it's `result.nr_views`, the `xa_` is removed from the fields on that class.